package.json

```diff
@@ -31,15 +33,20 @@
 	"license": "Apache-2.0",
 	"homepage": "https://github.com/siimon/prom-client",
 	"devDependencies": {
+		"benchmark": "^2.1.4",
+		"chalk": "^2.4.1",
+		"cli-table": "^0.3.1",
 		"eslint": "^5.6.0",
 		"eslint-plugin-node": "^7.0.1",
 		"eslint-plugin-prettier": "^2.1.2",
 		"express": "^4.13.3",
 		"husky": "^0.14.3",
 		"jest": "^23.6.0",
 		"lint-staged": "^7.0.0",
+		"lodash": "^4.17.11",
```

Comment:
Minor, and y'all can ignore this if you think it doesn't matter, but what do you think of depending only on `lodash/flatten`, and doing the rest with vanilla JS?

`forEach`, `map` and `keys` are easy with vanilla JS. `Object.values` is node 7+, so you'd have to use a `for...in` loop or similar.  `flatMap` could be done with `_.flatten(arr).map(...)`.

(❤️ fewer and smaller dependencies)

Reply:
👍